### PR TITLE
fix ci

### PR DESCRIPTION
**Pull request by graingert**
### Pull Request Checklist
- [x] Changes have tests
- [x] Authors have been added to [AUTHORS.md](https://github.com/dateutil/dateutil/blob/master/AUTHORS.md)
- [x] News fragment added in changelog.d. See [CONTRIBUTING.md](https://github.com/dateutil/dateutil/blob/master/CONTRIBUTING.md#changelog) for details

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ switch, and thus all their contributions are dual-licensed.
 - Thierry Bastian <thierryb@MASKED>
 - Thomas A Caswell <tcaswell@MASKED> (gh: @tacaswell) **R**
 - Thomas Achtemichuk <tom@MASKED>
+- Thomas Grainger <tagrain@gmail.com> (gh: @graingert)
```
Comment (graingert) [inline AUTHORS.md:113]: not sure if I should add `**R**` or `**D**` to this

Comment (graingert) [reply AUTHORS.md:113]: it's D

Comment (graingert) [inline AUTHORS.md:113]:
```suggestion
- Thomas Grainger <tagrain@gmail.com> (gh: @graingert) **D**
```

File: .github/workflows/validate.yml [hunk 2]
```diff
@@ -93,11 +93,7 @@ jobs:
...
         run: |
           python -m pip install --upgrade pip
-          if [[ $TOXENV == "tz" ]]; then
-            python -m pip install -U "tox<3.8.0"
-          else
-            python -m pip install -U tox
-          fi
+          python -m pip install -U tox
       - name: Run updatezinfo.py
         run: ./ci_tools/retry.sh python updatezinfo.py
```
[the remark this answers is not in the document]
Comment (graingert) [reply .github/workflows/validate.yml:98]: ah it's run only `if: matrix.toxenv == 'tox'` which is never true

**Comment by pganssle** [comment]
image

File: .github/workflows/validate.yml [hunk 3]
```diff
@@ -111,11 +104,12 @@ jobs:
         with:
           fetch-depth: 0
       - uses: actions/setup-python@v4
-      - uses: akaihola/darker@1.5.1
+      - uses: akaihola/darker@0fb2501a3f6c1b2d64976afa57885aeec0601182
+        # pinned due to unreleased fix: https://github.com/akaihola/darker/issues/489
```
Comment (m-aciek) [inline .github/workflows/validate.yml:108]: The fix has been released, fwiw https://github.com/akaihola/darker/blob/master/CHANGES.rst#fixed-1

**Pull request merged**